python/ray/serve/tests/test_cluster.py

```diff
@@ -130,8 +130,8 @@ def test_replica_startup_status_transitions(ray_cluster):
 
     @serve.deployment(version="1", ray_actor_options={"num_cpus": 2})
     class E:
-        def __init__(self):
-            ray.get(signal.wait.remote())
+        async def __init__(self):
+            await signal.wait.remote()
```

Comment:
i couldn't think of a case in serve that `ray.get(ref)` is more favorable compare to `await ref`, should we always use asyncio context for serve tests, and maybe even detect and recommend user to use async/await as warning message ?